src/app/component/dashboard/dashboard.component.html

```diff
@@ -643,6 +643,13 @@
                                     language["addHoliday"]
                                     }}</span></a>
                               </li>
+
+                              <li class="{{ selectedNodeModel['add_holiday'] }}">
```

Comment:
Why we have for this selectedNodeModel['add_holiday']?
It's not good.
It's need to be selectedNodeModel['manageClinics']